packages/did-ethr-resolver/src/implementations/operator.ts

```diff
@@ -47,19 +47,23 @@ export class Operator extends Resolver implements IOperator {
 
   private address?: string;
 
-  protected readonly _owner: IdentityOwner;
+  // protected readonly _owner: IdentityOwner;
+  private _owner: IdentityOwner;
 
   /**
  * @param { IdentityOwner } owner - entity which controls document updatable by this operator
  */
-  constructor(owner: IdentityOwner, settings: RegistrySettings) {
-    super(owner.provider as providers.Provider, settings);
+  //  constructor(owner: IdentityOwner, settings: RegistrySettings) {
+  constructor(privateKey: string, settings: RegistrySettings, providerUrl?: string) {
```

Comment:
Why is `providerUrl` optional? Please explain in param comment

Reply:
I like the removal of the `ethers` objects from interface. A note that we also need to add EIP-1193 provider (i.e. web3Provider) as option.